src/Uno.UI.FluentTheme.v1/Resources/Version1/PriorityDefault/MediaTransportControls_themeresources.xaml

```diff
@@ -778,7 +778,7 @@
                                         </AppBarButton.Icon>
                                     </AppBarButton>
                                 </Border>
-                                <Border x:Name="MediaTransportControls_Command_Border" Grid.Column="1" Grid.Row="2">
```

Comment:
This feels like something that may be overwritten by a future fluent styles update. @agneszitte @MartinZikmund would you know if this is something that was changed in a recent fluent styles update in WinUI ?

Reply:
@rafael-rosa-knowcode what was the intent behind the need to add the additional `HorizontalAlignment="Center"`?

Since I am not seeing it in the WinUI common styles:
- https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/CommonStyles/MediaTransportControls_themeresources.xaml
- https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/CommonStyles/MediaTransportControls_themeresources_v1.xaml

**I am pretty sure that we would need to do the opposite and understand why one of the Panel or Control is not behaving the same on the other platforms compared to Windows with the exact same Style**

![image](https://github.com/unoplatform/uno/assets/16295702/00ec6b44-9019-4d25-9e6c-6c65a626c8a1)

Reply:
@jeromelaban , in this case, should we update the issue in this PR?
CI is crashing in UI Tests Screenshots Comparison.
Because the controls are centralized now.
Will the changes be overwritten?
Do I need to look at the Tests Screenshots Comparison?

Reply:
@agneszitte , sorry. Just now i see your comment.

I change to the center to be like the windows, you show in the expected.


And now is like:


this is the minimum (Wasm, Wasm Generic, Windows on Gallery)

![image](https://github.com/unoplatform/uno/assets/116665025/aad3f004-20d5-46ba-b38e-3bba946f3aed)

And this is the full (Wasm and Windows on Gallery)

![image](https://github.com/unoplatform/uno/assets/116665025/a71b9ef6-b7fd-4f3f-a3ea-a34dfc5e560c)

But you are correct. Now i see. 
We need to make the same style [MediaTransportControls_themeresources.xaml](https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/CommonStyles/MediaTransportControls_themeresources.xaml) looks like the reference.
Will try to list the differences and fix.